cmd/support_archive/config.go

```diff
@@ -13,3 +13,5 @@ const ManifestsFileExtension = ".yaml"
 const CRDKindName = "CustomResourceDefinition"
 const ValidatingWebhookConfigurationKind = "ValidatingWebhookConfiguration"
 const MutatingWebhookConfigurationKind = "MutatingWebhookConfiguration"
+
+const NumEvents = 10
```

Comment:
```suggestion
const NumEvents = 300
```
or better let's make it configurable via env variable or so.